dockorm/tests/test_container.py

```diff
@@ -287,5 +287,6 @@ def test_build_failed_pull(capsys):
     assert (
         stdout[1] ==
         "pull access denied for dockorm_fake_org/dockorm_fake_image, "
-        "repository does not exist or may require 'docker login'"
+        "repository does not exist or may require 'docker login': "
+        "denied: requested access to the resource is denied"
```

Comment:
This looks good to me. (Have you run the tests to confirm you have the spacing and such right?)

I'm not sure what about the environment changed, but this is required for master even. Maybe ECR changed its reply?

Reply:
My guess ECR changed?

@richafrank I copy/pasted this direct from the test output of the project this was failing in. We could also make this less exact:

`assert("pull access denied for dockorm_fake_org/dockorm_fake_image" in stdout[1])`

Thoughts?

Reply:
Ah right - The latter sounds more robust to me.